### ISSUE TITLE
TS Server fatal error:  Cannot read property 'indexOf' of undefined

### ISSUE THREAD

**Issue by smcenlly** [microsoft/vscode#116384]
Issue Type: <b>Bug</b>

❗️❗️❗️ Please fill in the sections below to help us diagnose the issue ❗️❗️❗️

**TypeScript Version:** 4.1.4

**Steps to reproduce crash**
1.
2.
3.

**TS Server Log**
❗️Server logging disabled. To help us fix crashes like this, please enable logging by setting:
```json
"typescript.tsserver.log": "verbose"
```
After enabling this setting, future crash reports will include the server log.

**TS Server Error Stack**
Server: `semantic`
```
TypeError: Cannot read property 'indexOf' of undefined
    at Object.parsePackageName (c:\Users\Simon McEnlly\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:39472:30)
    at ConfiguredProject.Project.enablePlugin (c:\Users\Simon McEnlly\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:151962:24)
    at ConfiguredProject.enablePluginsWithOptions (c:\Users\Simon McEnlly\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:152477:30)
    at ProjectService.loadConfiguredProject (c:\Users\Simon McEnlly\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:154206:25)
    at ProjectService.createAndLoadConfiguredProject (c:\Users\Simon McEnlly\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:154145:22)
    at ProjectService.createLoadAndUpdateConfiguredProject (c:\Users\Simon McEnlly\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:154150:36)
    at ProjectService.assignProjectToOpenedScriptInfo (c:\Users\Simon McEnlly\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:155055:44)
    at c:\Users\Simon McEnlly\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:155395:97
    at Object.flatMap (c:\Users\Simon McEnlly\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:513:25)
    at ProjectService.applyChangesInOpenFiles (c:\Users\Simon McEnlly\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:155395:41)
    at Session.handlers.ts.Map.ts.getEntries._a.<computed> (c:\Users\Simon McEnlly\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:156413:46)
    at c:\Users\Simon McEnlly\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:158286:88
    at IOSession.Session.executeWithRequestId (c:\Users\Simon McEnlly\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:158277:28)
    at IOSession.Session.executeCommand (c:\Users\Simon McEnlly\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:158286:33)
    at IOSession.Session.onMessage (c:\Users\Simon McEnlly\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:158310:35)
    at Interface.<anonymous> (c:\Users\Simon McEnlly\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:160520:27)
    at Interface.emit (events.js:315:20)
    at Interface._onLine (readline.js:329:10)
    at Interface._normalWrite (readline.js:474:12)
    at Socket.ondata (readline.js:186:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    at readableAddChunk (_stream_readable.js:271:9)
    at Socket.Readable.push (_stream_readable.js:212:10)
    at Pipe.onStreamRead (internal/stream_base_commons.js:186:23)
```

VS Code version: Code 1.53.1 (5d424b828ada08e1eb9f95d6cb41120234ef57c7, 2021-02-08T23:29:42.785Z)
OS version: Windows_NT x64 10.0.19041

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz (12 x 3696)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.82GB (15.80GB free)|
|Process Argv|--crash-reporter-id 3bd67c21-d7ca-41fe-aa42-3c853e81c1bd|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (28)</summary>

Extension|Author (truncated)|Version
---|---|---
better-comments|aar|2.1.0
vscode-opennewinstance|chr|0.0.7
vscode-eslint|dba|2.1.14
xml|Dot|2.5.1
gitlens|eam|11.2.1
EditorConfig|Edi|0.16.4
prettier-vscode|esb|5.9.1
vscode-scss|mrm|0.9.0
vscode-docker|ms-|1.9.1
python|ms-|2021.1.502429796
jupyter|ms-|2020.12.414227025
remote-containers|ms-|0.158.0
remote-ssh|ms-|0.63.0
remote-ssh-edit|ms-|0.63.0
remote-wsl|ms-|0.53.4
vscode-remote-extensionpack|ms-|0.20.0
vscode-js-profile-flame|ms-|0.0.14
vsliveshare|ms-|1.0.3629
debugger-for-chrome|msj|4.12.12
vscode-react-native|msj|1.3.0
typescript-hero|rbb|3.0.0
tabnine-vscode|Tab|3.2.7
open-in-browser|tec|2.0.0
dingo-vscode|Wal|1.0.8
quokka-vscode|Wal|1.0.342
wallaby-vscode|Wal|1.0.263
blade-runner|yuk|1.0.0
markdown-all-in-one|yzh|3.4.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492cf:30211402
vstry914:30250197
pythonvsdeb440:30248342
pythonvsded773:30248341
vstes627:30244334
pythonvspyt600cf:30251589

```

</details>

**Comment by smcenlly** [comment]
Please note that this was a breaking change in the latest version of TypeScript Language Service/vscode.

Fixed in my project with tsconfig.json change:

```diff
{
  "compilerOptions": {
    "module": "commonjs",
    "target": "es6",
    "sourceMap": false,
    "removeComments": false,
    "moduleResolution": "node",
    "lib": [ "es2017" ],
    "outDir": "./build/output",
    "allowJs": true,
    "allowSyntheticDefaultImports": true,
    "baseUrl": "src",
    "paths": {
      "*": ["*"]
    },
    "plugins": [
      {
+       "name": "ts-transform-paths",
        "transform": "@zerollup/ts-transform-paths",
        "exclude": ["*"]
      }
    ]
  },
  "include": [
    "**/*.ts",
    "**/*.js"
  ],
  "exclude": [
    "build.js",
    "build-clean.js",
    "wallaby.js",
    "node_modules",
    "build",
    "**/__mocks__/*"
  ]
}
```

**Comment by gjsjohnmurray** [comment]
/duplicate of #116219 and will be fixed in upcoming 1.53.2
